store/piecestore/storage/const.go

```diff
@@ -10,6 +10,8 @@ const (
 	MinioStore = "minio"
 	// DiskFileStore defines storage type for file
 	DiskFileStore = "file"
+	// LdfsStore defines storage type for ldfs
+	LdfsStore = "ldfs"
 	// MemoryStore defines storage type for memory
```

Comment:
Is there any item needed for IAMType configuration?

Reply:
In case of GF we don't need additional authentication. Because we use a whitelist to restrict the IPs that can access LDFS.